01-python/01-python.md

```diff
@@ -38,12 +38,11 @@ for the fruits of the language are manyfold, and in your hands lay but a taste.
 -   isinstance and hasattr
 -   mixin
 -   whl file
--   unit test vs integration test
--   fixtures
--   tox
+
     
 ### Final Exercise:
 - Flutter over [w3schools' Python Tutorial + File Handling](https://www.w3schools.com/python/default.asp) and make sure you are familiar with all concepts
 - Do the w3schools [python quiz](https://www.w3schools.com/python/python_quiz.asp)
 - Let your instructor choose for you a random 10 exercises from [w3resource](https://www.w3resource.com/python-exercises/)
+- Write a simple Fastapi server. Use gitlab to manage the code.
```

Comment:
In the environment or www? Also add basic requirements such as access an api have 3 endpoints use different http methods etc